homeassistant/components/rhasspy/default_settings.py

```diff
@@ -0,0 +1,351 @@
+                    T(
+                        "(is | are) [the|a|an] ({{ speech_name }}){name:{{ friendly_name }}} off"
+                    )
+                ]
+            }
+        ],
+        INTENT_IS_COVER_OPEN: [
+            {
+                KEY_COMMAND_TEMPLATES: [
+                    T(
+                        "(is | are) [the|a|an] ({{ speech_name }}){name:{{ friendly_name }}} open"
+                    )
+                ],
+            }
+        ],
+        INTENT_IS_COVER_CLOSED: [
+            {
+                KEY_COMMAND_TEMPLATES: [
+                    T(
+                        "(is | are) [the|a|an] ({{ speech_name }}){name:{{ friendly_name }}} closed"
+                    )
+                ]
+            }
+        ],
+        INTENT_IS_DEVICE_STATE: [
+            {
+                KEY_COMMANDS: ["is it sunset", "has the sun set [yet]"],
+                KEY_DATA: {"name": "sun", "state": "below_horizon"},
+            }
+        ],
+        INTENT_TRIGGER_AUTOMATION: [
+            {
+                KEY_COMMAND_TEMPLATES: [
+                    T(
+                        "(run | execute | trigger) [program | automation] ({{ speech_name }}){name:{{ friendly_name }}}"
+                    )
+                ]
+            }
+        ],
+        INTENT_TRIGGER_AUTOMATION_LATER: [
+            {
+                KEY_COMMAND_TEMPLATES: [
+                    T(
+                        "(run | execute | trigger) [program | automation] ({{ speech_name }}){name:{{ friendly_name }}} (in | after) <SetTimer.time_expr>"
+                    ),
+                    T(
+                        "(in | after) <SetTimer.time_expr> (run | trigger) [program | automation] ({{ speech_name }}){name:{{ friendly_name }}}"
+                    ),
+                ]
+            }
+        ],
+        INTENT_SET_TIMER: [
+            {
+                KEY_COMMANDS: [
+                    "two_to_nine = (two:2 | three:3 | four:4 | five:5 | six:6 | seven:7 | eight:8 | nine:9)",
+                    "one_to_nine = (one:1 | <two_to_nine>)",
+                    "",
+                    "teens = (ten:10 | eleven:11 | twelve:12 | thirteen:13 | fourteen:14 | fifteen:15 | sixteen:16 | seventeen:17 | eighteen:18 | nineteen:19)",
+                    "",
+                    "tens = (twenty:20 | thirty:30 | forty:40 | fifty:50)",
+                    "one_to_nine = (one:1 | <two_to_nine>)",
+                    "",
+                    "one_to_fifty_nine = (<one_to_nine> | <teens> | <tens> [<one_to_nine>])",
+                    "two_to_fifty_nine = (<two_to_nine> | <teens> | <tens> [<one_to_nine>])",
+                    "hour_half_expr = (<one_to_nine>{hours} and (a half){{minutes:30}})",
+                    "hour_expr = (((one:1){hours}) | ((<one_to_nine>){hours}) | <hour_half_expr>) (hour | hours)",
+                    "minute_half_expr = (<one_to_fifty_nine>{minutes} and (a half){{seconds:30}})",
```

Comment:
I don't know if it's intended, but it caused an error when I copied these rules in my Rhasspy instance manually: double curly brackets around minutes:30 and seconds:30 are wrong (they produce an error in my instance).